gunicorn/pidfile.py

```diff
@@ -66,7 +66,10 @@ def validate(self):
             return
         try:
             with open(self.fname, "r") as f:
-                wpid = int(f.read() or 0)
+                try:
+                    wpid = int(f.read())
+                except ValueError:
+                    wpid = 0
```

Comment:
Might be clearer to simply return rather than relying on the subsequent if statement.